README.md

```diff
@@ -59,6 +59,13 @@ let args = {
 vscode.commands.executeCommand("graphviz-interactive-preview.preview.beside", args)
 ```
 
+## Maintainers
+
```

Comment:
The gifs which show the extension should be updated

Reply:
updated the readme: preview 👉 https://github.com/tintinweb/vscode-interactive-graphviz/blob/f7f1d2fdcb9d4f46d8f3095b017feb9286a33698/README.md

wdyt @bigbug ?